src/components/aboutMe.js

```diff
@@ -20,7 +20,7 @@ const AboutMe = () => (
       I am a Software Engineer based out of Karlsruhe, Germany. Born and raised
       in Upstate New York, I sold everything and moved to Europe two years ago.
     </p>
-    <Link className="learn-more" to="/about">
+    <Link className="learn-more" to="/about" aria-label="Learn more about Emma Bostian">
```

Comment:
We don't need aria-label because the text is visible in the UI :)

Reply:
No worries :) Sorry I'm currently still learning a lot on the accessibility side!

It had come from my research into things like this - https://www.nngroup.com/articles/learn-more-links/ - where when people using screen readers may just have the list of links read out to them, without the context of the content preceding to the link. Still trying to gain knowledge in this area though so thanks for the tip!